src/pybind/mgr/status/module.py

```diff
@@ -233,6 +233,10 @@ def handle_fs_status(self, cmd):
             output += rank_table.get_string()
             output += "\n" + pools_table.get_string() + "\n"
 
+        if not output:
```

Comment:
I'd suggest changing this to `if not output and fs_filter is not None`, so that if someone is just running "fs status" on a system with no filesystems they don't get this message.